src/schema/rules/sidecars/motion.yaml

```diff
@@ -0,0 +1,74 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+---
+motionGeneric:
+  selectors:
+    - modality == "motion"
+    - datatype == "motion"
+    - suffix == "motion"
+  fields:
+    TaskName:
+      level: required
+
+motionRecommended:
+  selectors:
+    - modality == "motion"
+    - datatype == "motion"
+    - suffix == "motion"
+  fields:
+    InstitutionName: recommended
+    InstitutionAddress: recommended
+    InstitutionalDepartmentName: recommended
+    Manufacturer: recommended
+    ManufacturersModelName: recommended
+    SoftwareVersions: recommended
+    TaskDescription: recommended
+    Instructions: recommended
+    DeviceSerialNumber: optional
+    ExternalSoftwareVersions: optional
+    Manufacturer: optional
+    ManufacturersModelName: optional
+    SoftwareVersions: optional
```

Comment:
CI says you have those as duplicates:

https://github.com/bids-standard/bids-specification/actions/runs/4127444756/jobs/7130662670#step:5:13

Reply:
Fixed it.